frontend/src/metabase/public/containers/PublicDashboard.jsx

```diff
@@ -146,7 +147,11 @@ class PublicDashboard extends Component {
         >
           {() => (
             <DashboardContainer>
-              <DashboardTabs />
+              <DashboardTabs
+                slug={this.props.params.tabSlug}
+                pathname={this.props.location.pathname}
+              />
+              <Separator />
```

Comment:
This is not related to URLs, but I forgot to put the underline back after it was removed from the `TabRow` component in https://github.com/metabase/metabase/pull/31359